src/Ninja/G1CP/Content/Fixes/Session/fix030_SilasTrade.d

```diff
@@ -2,6 +2,10 @@
  * #30 Silas trades only one time
  */
 func int G1CP_030_SilasTrade() {
-    var int symbId; symbId = G1CP_GetInfoInstId("DIA_Silas_Trade");
-    return (G1CP_ReplaceAssignInt(symbId, 0, "C_Info.permanent", 0, 0, 1) > 0);
+    const string infoName = "DIA_Silas_Trade";
+    const string infoProp = "permanent";
+    const int originalInt = 0;
+    const int correctedInt = 1;
+
+    return (G1CP_ChangeInfoIntVar(infoName, infoProp, originalInt, correctedInt) > 0);
```

Comment:
I would prefer to remove the constants here, as they do not add much value - in terms of redundancy or readability. The line of code below is already very clear to me. This is my impression; Let's discuss!
Additionally, the `> 0` is not necessary, because the defined function returns exclusively `TRUE` or `FALSE`

```D
return G1CP_ChangeInfoIntVar("DIA_Silas_Trade", "permanent", 0, 1);
```

Reply:
Agreed